source/Files.h

```diff
@@ -66,6 +66,9 @@ class Files {
 	static void Write(const std::string &path, const std::string &data);
 	static void Write(FILE *file, const std::string &data);
 
+	// Opens the plugins folder inside the configuration directory in a separate window.
+	static void OpenGlobalPluginFolder();
```

Comment:
This actually opens the user-specific plugin directory, not the systemwide plugin directory (which is located in the install directory), so the function name is not quite correct.

Comment should be e.g. `// Open this user's plugins directory in their native file explorer.`